tests/navigation.rs

```diff
@@ -147,3 +148,30 @@ fn can_navigate_using_breadcrumbs(
 
     Ok(())
 }
+
+#[rstest]
+#[case(server(&["--default-sorting-method", "date", "--default-sorting-order", "desc"]))]
+#[case(server(&["--default-sorting-method", "date", "--default-sorting-order", "asc"]))]
+/// We can specify the default sorting order
+fn can_specify_default_sorting_order(#[case] server: TestServer) -> Result<(), Error> {
+    let slash = String::from("/");
```

Comment:
I'm not sure how I feel about this since we only really need it at a single place. Maybe just create this where it needs to be and remove this?